Diary/src/main/java/com/microsoft/device/display/samples/diary/ui/theme/Theme.kt

```diff
@@ -0,0 +1,31 @@
+package com.microsoft.device.display.samples.diary.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val LightColorPalette = lightColors(
+    primary = Purple500,
+    primaryVariant = Purple700,
+    secondary = Teal200
+)
+
+@Composable
+fun DiaryTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
+    val colors = if (darkTheme) {
+        LightColorPalette
+    } else {
+        LightColorPalette
+    }
```

Comment:
if you don't plan on using a dark theme, you can just delete this if and the dark theme boolean and set the colors to the light color palette